int32 student_id
---
string name
float32 score
